tests/src/JIT/Regression/JitBlue/GitHub_19454/Github_19454.cs

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+// This test was extracted from the corefx System.Numerics.Vectors tests,
+// and was failing with minOpts because a SIMD12 was being spilled using
+// a 16-byte load, but only a 12-byte location had been allocated.
```

Comment:
I suspect that this was a cut-and-paste error, and should be updated?

Reply:
Yeah, let me fix that.